Makefile

```diff
@@ -0,0 +1,91 @@
+
+VERSION := 0.1.0
+
+# Repository
+SRC_DIR := src
+
+PARSER_REPO_URL   ?= $(shell git -C $(SRC_DIR) remote get-url origin)
+$(eval PARSER_NAME=$(shell basename $(PARSER_REPO_URL) | cut -d '-' -f3 | sed 's#.git##' ))
+UPPER_PARSER_NAME := $(shell echo $(PARSER_NAME) | tr a-z A-Z)
+
+# install directory layout
+PREFIX ?= /usr/local
```

Comment:
So is this only going to work on Unix-based systems? I'm fine avoiding the yak-shaving if that's the only place you need it, but want to check.

Reply:
yes, it's not ready for Windows at this point.